config/permissions.go

```diff
@@ -107,26 +123,6 @@ var (
 			Entity: "operator",
```

Comment:
Would be possible to not rewrite these again, just inherits like  `...ReadonlyPermissions` ?